recipes/virmet/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: virmet
+  version: "v1.0"
+
+source:
+  url: https://github.com/ozagordi/VirMet/archive/v1.0.tar.gz
+  md5: 3ac2a7fb32fb57dcd061f68b2901e832
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - biopython
+    - blast >=2.3
+    - bwa
+    - entrez-direct
+    - pandas
+    - prinseq
+    - python
+    - samtools >=1.3
+    - seqtk
+
+about:
+  home: http://virmet.readthedocs.io
+  summary: 'A pipeline for viral metagenomics'
+  license: MIT
+  license_file:
```

Comment:
Perhaps remove or comment out `license_file` since the repo/tarball does not seem to include one.